assets/timeTable.js

```diff
@@ -707,10 +755,10 @@ jQuery(document).ready(function ($) {
 </div>`;
           },
           option: function (item, escape) {
-            return `<div><span>${escape(item.text)} <span><i class="fa fa-angle-right fa-xs"></i> ${escape(item.projectName)} <small>(${escape(item.value)})</small> <small style="float: right;">${item.editorId === pluginSettings.userId ? '<i class="your-task far fa-user" title="To-do is assigned to you"></i>' : ""}(${escape(item.type)})</small></span></span></div>`;
+            return `<div><span>${escape(item.text)} <span><i class="fa fa-angle-right fa-xs"></i> ${escape(item.projectName)} <small>(${escape(item.value)})</small> <small style="float: right;">${item.editorId === pluginSettings.userId ? '<i class="your-task far fa-user" title="To-do is assigned to you"></i>' : ""}${item.type.toLowerCase() !== "task" ? `(${escape(item.type)})` : ""}</small></span></span></div>`;
```

Comment:
what is this?

Reply:
It's the html definition of the options in the to-do search dropdown. The addition lowercases the type name, because we had instances of to-dos with the type "Task" and "task", causing problems cause letter case was not taken into consideration.

Reply:
And! We only want to display types not equal to "task". 

It clutters a lot with "task" written on every to-do, so we figured that it would be better just to print it on to-dos with another type than task.